oci-info/action/args.py

```diff
@@ -0,0 +1,164 @@
+# SPDX-FileCopyrightText: 2024 Greenbone AG
+#
+# SPDX-License-Identifier: AGPL-3.0-or-later
+
+"""
+Module to parse command-line arguments.
+"""
+
+from argparse import ArgumentParser, Namespace
+from typing import Optional, Sequence
+
+import shtab
+
+
+def parse_args(args: Optional[Sequence[str]] = None) -> Namespace:
+    """
+    Parse command-line arguments.
+
+    Args:
+        args (Optional[Sequence[str]]): List of command-line arguments.
```

Comment:
Not necessary to add the type to the arguments and return values. If you use for example sphinx for generating docs from the docstring, it reads the types from the code annotations automatically.
```suggestion
        args: List of command-line arguments.
```